cppcache/integration/test/PartitionRegionOpsTest.cpp

```diff
@@ -15,6 +15,8 @@
  * limitations under the License.
  */
 
+#include <stdio.h>
```

Comment:
Why is this header necessary?

Reply:
Good catch, it is not needed. I forgot to remove it when preparing the PR.